### DISCUSSION TITLE
About Loki logs

### DISCUSSION THREAD

**Discussion by microgiantya** [sipcapture/homer-app#550] [Q&A]
Hi!

The default logQL query, for example, with the asterisk log line NoOp(${SIP_HEADER(CALL-ID)}) looks like the image:

![homer-app_loki](https://github.com/sipcapture/homer-app/assets/81983258/60161523-3ad0-4edb-b429-09ec9bb63f25)

How can be shown all log lines?

Thanx!

**Comment by microgiantya** [comment]
Please, note !))

**Comment by lmangani** [comment]
I'm sorry the question is unclear. Are you asking how to return more log lines or how to change the template query?

**Reply by mcrgnt** [reply]
Hi, Imangani! Thanks for reply.
Actually i what to know: how i can get all needable log lines. For example using asterisk's CID [C-00000000]. In my opinion, if we can use only %callid%, that is at least two queries. First - query for log lines which includes %callid% and second - stream selector using extracted label(s) (promtail).

Maybe exists another approach?
Thx ))

**Comment by mcrgnt** [comment]
Any updates?

**Comment by hrhashmi** [comment]
@mcrgnt as Homer does not have channel ID information, you can copy the channel ID from the first line and replace it with Call-ID in LogQL Query and click `RunQuery` again. It will show you the other logs line with the same channel ID. 

Or 

Add ChannelID as an SIP Header then configure [custom header](https://github.com/sipcapture/homer/wiki/Custom-Header-Searching-and-Correlation) for Channel-ID and use that Header directly in heplify-server configuration.

**Comment by mcrgnt** [comment]
@hrhashmi, thx for reply
